modules/apps/remote-app/remote-app-client-js/src/main/resources/META-INF/resources/js/index.ts

```diff
@@ -36,6 +36,20 @@ type ClientOptions = {
 	debug: boolean;
 };
 
+/**
+ * Properties accessible via the `get()` method. The remote-app-support-web
+ * module will reject requests for properties outside of this list.
+ */
+type GettableProperties =
```

Comment:
How would one consume these definitions when writing a remote app? 

We'll need to publish these or `d.ts` files to match, I guess?

Reply:
> We'll need to publish these or `.d.ts` files to match, I guess?

Yep.